api-tests/src/steps/ipv-steps.ts

```diff
@@ -155,13 +158,32 @@ Then(
       isPageResponse(this.lastJourneyEngineResponse),
       `got a ${describeResponse(this.lastJourneyEngineResponse)}`,
     );
-    assert.equal("page-ipv-reuse", this.lastJourneyEngineResponse.page);
 
     const provenIdentity = await getProvenIdentityDetails(this.ipvSessionId);
 
-    assert.ok(
-      doProvenDetailsMatchIdentity(provenIdentity, this.identity),
-      "Proven identity and current identity do not match.",
+    const expectedAddresses = this.identity[addressCredential];
+    assert.deepEqual(
+      provenIdentity.addresses,
+      expectedAddresses,
+      "Addresses do not match.",
+    );
+
+    const expectedBirthDate =
+      this.identity[identityCredential].birthDate &&
+      this.identity[identityCredential].birthDate[FIRST].value;
```

Comment:
We're in the wonderful world of TS now, you can use null-chaining expressions, e.g.:
```
const expectedBirthDate = this.identity[identityCredential].birthDate?.[0].value;
```
(I'm also not totally sold on `FIRST = 0` as a useful constant, but it's fine)